### PR TITLE
Model Zoo project submission - SSD tensorflow

### PR DESCRIPTION
**Pull request by akhilagrawal1001**
The model is not complete yet, contain model architecture, loss function, data generator only as of now
Akhil Agrawal
200076

### CODE REVIEW

**Comment by akhilagrawal1001** [comment]
Add code for model training, loaded data, but training fails ( loss increase with time )

**Comment by AthaSSiN** [comment]
1. Edit pull request title to display the name of the model and your framework.
2. Use present-tense, imperative-style for writing commit messages: "Add code for model training, load data ..."
3. Code quality and documentation is proper.
4. Explain the purpose of the functions that you have created in the utils using comments, for better understanding.

**Comment by namangup** [comment]
Fix the training issue. Add loss graph, metrics after training.
